### ISSUE TITLE
README link to Free Law Ferret is dead

### ISSUE THREAD

**Issue by troglodite2** [freelawproject/reporters-db#165]
The link to the Firefox browser leads to a 404 page.

**Comment by mlissner** [comment]
I can't find it online. We can just remove the reference. @fbennett used to maintain the Ferret, but it's been many years.

**Comment by fbennett** [comment]
Yes, the right move would be to remove the reference. There are better
tools for the job out there, and I have more important, or at least more
pressing, things on my plate.

**Comment by mlissner** [comment]
Thanks Frank! I thought you'd say as much. I hope you're well.
